Mathlib/SetTheory/Ordinal/Basic.lean

```diff
@@ -1234,6 +1234,10 @@ theorem ord_eq_zero {a : Cardinal} : a.ord = 0 ↔ a = 0 :=
 theorem ord_eq_one {a : Cardinal} : a.ord = 1 ↔ a = 1 :=
   ord_injective.eq_iff' ord_one
 
+@[simp]
+theorem omega0_le_ord {a : Cardinal} : ω ≤ a.ord ↔ ℵ₀ ≤ a := by
+  rw [← ord_aleph0, ord_le_ord]
```

Comment:
Do we have lemmas with `ω` on the other side of the inequality? Do we have lemmas about equalities? If no, then please add them.

Reply:
The "lemma" `a.ord = ω ↔ a = ℵ₀` is false. But `a.ord < ω ↔ a < ℵ₀` is true and trivially provable from this one, so I'll add it.

Reply:
I've just pushed a proof of this lemma.

Reply:
Oh you're right, it's `a.card = ℵ₀ ↔ a = ω` which is wrong.